README.md

```diff
@@ -172,3 +172,15 @@ It is acceptable to use the following section divider when it aids clarity. For
 ```python
 # -----------------------------------------------------------------------------
 ```
+
+## Import styles
```

Comment:
this should probably be a bit earlier in the guide... maybe before "Preferred tools"